service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationRoutes.java

```diff
@@ -203,8 +202,17 @@ private void configureMasProcessing() {
             auditProcessor(orderExamRouteId, "There is insufficient evidence. Ordering an exam"))
         .process(masOrderExamProcessor)
         .log("MAS Order Exam response: ${body}")
-        // Upload PDF only if SufficientForFastTracking is either true or false
+        // Upload PDF but catch errors since exam was ordered and continue
+        .doTry()
         .to(ENDPOINT_UPLOAD_PDF)
+        .endDoTry()
+        .doCatch(BipException.class)
+        // Mas Complete Processing code expects this to be the body of the message
+        .setBody(simple("${exchangeProperty.payload}"))
+        .wireTap(ENDPOINT_NOTIFY_AUDIT) // Send error notification to slack
+        .onPrepare(
+            slackEventProcessor(orderExamRouteId, "PDF upload failed after exam order requested."))
+        .endDoCatch()
```

Comment:
Based on this "Instead of end() you can use endDoTry() to end and return back to the try …​ catch scope." from camel documentation you need end I think. Exam order end to end tests are failing because processing stops right after the pdf is uploaded.  I do not think you need ".setBody(simple("${exchangeProperty.payload}"))" but I have not tested that yet. I will start trying with end on my end.

Reply:
The set body is for the MAS complete route that needs it. it's not set to the body at that point. I forget what is, but it's not that. It definitely fails without it.

As far as the end vs endDoTry() that's....confusing. It was my understanding endDoTry was only the end ofthe try block and end() ends the route itself.

Reply:
that said if end() not endDoTry is right the lighthouse code may also be wrong i'll switch back to this branch in a moment.